package-lock.json

```diff
@@ -0,0 +1,4023 @@
+{
```

Comment:
It is a reasonable practice to add package-lock.json to the repo, but please do it as a separate commit.

Reply:
I reset and added package-lock.json as a separate commit